Library/Homebrew/cask/cask_loader.rb

```diff
@@ -228,18 +228,25 @@ def load(config:)
         json_cask = @from_json || Homebrew::API::Cask.all_casks[token]
         cask_source = JSON.pretty_generate(json_cask)
 
-        json_cask = Homebrew::API.merge_variations(json_cask).deep_symbolize_keys
+        json_cask = Homebrew::API.merge_variations(json_cask).deep_symbolize_keys.freeze
```

Comment:
We access this a million times here but don't actually need to modify it so I think freezing it makes sense. Personal preference kind of thing though.